DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/os/linux/linux_sysctl_impl.py

```diff
@@ -0,0 +1,41 @@
+from dent_os_testbed.lib.os.linux.linux_sysctl import LinuxSysctl 
+class LinuxSysctlImpl(LinuxSysctl):
+    """
+        system kernel attributes manager
+    """
+    def format_get(self, command, *argv, **kwarg):
+        """
+        Get the attribute value
+        
+        """
+        params = kwarg["params"]
+        cmd = 'sysctl -n {} {}'.format(command, params["variable"])
+        ############# Implement me ################
+        
+        return cmd
+        
+    def parse_get(self, command, output, *argv, **kwarg):
+        """
+        Get the attribute value
+        
+        """
+        return output.strip()
```

Comment:
I think that parsing `sysctl` output into a dict would be much more useful (see `linux_ethtool_impl.py` for reference).
For a single value it would be an overkill, but `sysctl` can handle multiple values at once and storing all of the variables as key-value pairs would be very handy:
```python
out = await Sysctl.get(input_data=[{dent: [{"variable": "net.ipv4"}]}], parse_output=True)
print(out[0][dent]["parsed_output"])
# {
# ...
#     "net.ipv4.conf.all.accept_local": "0",
#     "net.ipv4.conf.all.accept_redirects": "0",
#     "net.ipv4.conf.all.accept_source_route": "0",
#     "net.ipv4.conf.all.arp_accept": "0",
#     "net.ipv4.conf.all.arp_announce": "0",
# ...
# }
```